pom.xml

```diff
@@ -82,7 +82,7 @@
         <h2.version>2.1.214</h2.version>
         <hibernate-orm.version>6.2.1.Final</hibernate-orm.version>
         <hibernate.c3p0.version>${hibernate-orm.version}</hibernate.c3p0.version>
-        <infinispan.version>14.0.8.Final</infinispan.version>
+        <infinispan.version>14.0.9.Final</infinispan.version>
```

Comment:
@martin-kanis @mhajas This change looks safe to me but I'd like to have your opinion here.

Reply:
LGTM